### ISSUE TITLE
spikeforest recordings repo

### ISSUE THREAD

**Issue by magland** [spikodrome/administrative#12]
@yarikoptic and all

I made a repo containing the spikeforest recordings: https://github.com/flatironinstitute/spikeforest_recordings

I also have tools for online vis of these. And one of the next steps will be to make containerized spike sorting. I'll follow up on these topics in other gh issues.

**Comment by yarikoptic** [comment]
Great!  I will look into it in greater detail when I come back from current trip.
I might produce for myself a datalad'ified version (i.e. look into adding a git annex special remote to fetch using kachery those URLs like "sha1dir://fb52d510d2543634e247e0d2d1d4390be9ed9e20.synth_magland/datasets_noise20_K20_C4/006_synth/raw.mda" and placing them alongside the .json files.  E.g. for 
`./recordings/SYNTH_MAGLAND/synth_magland_noise20_K20_C4/006_synth.json` which fetches data from above url, have `006_synth.mda` nearby.   Or do you think it is a bad idea?

**Comment by magland** [comment]
Yes, I think that would be a good idea. Especially since data on kachery is not really archived in that it is not guaranteed to be there in, say, a month, a year, ten years.

**Comment by magland** [comment]
There is now a top-level file in the repo that contains a pointer to the entire spikeforest collection of recordings in a single json file.

It is here: https://github.com/flatironinstitute/spikeforest_recordings/blob/master/recordings/studysets

You can then copy that reference into the query parameter of a webapp like so: http://132.249.245.246:24348/app/spikeforest?path=sha1://d52cfae9e066334be3f21ffb0347a7b882d45818/studysets.json

which allows interactive browsing of these recordings. If you drill down you will find the timeseries view and other widgets such as autocorrelograms.
